Tests/PSGetFindModule.Tests.ps1

```diff
@@ -55,6 +56,134 @@ function SuiteCleanup {
     $null = Import-PackageProvider -Name PowerShellGet -Force
 }
 
+Describe FindModuleTests -Tags "TDD" {
+    
+    BeforeAll {
+        SuiteSetup
+    }
+
+    AfterAll {
+        SuiteCleanup
+    }
+
+    
+    # Purpose: Validate Find-Module (on a module with stable and prerelease versions)
+    #
+    # Action: Find-Module
+    #
+    # Expected Result: Find-Module should return the latest stable version, not the latest prerelease version of the module
+    #
+    It FindModuleReturnsLatestStableVersion {
+        $psgetModuleInfo = Find-Module -Name $script:PrereleaseTestModule -Repository Local
```

Comment:
I think reviewing the test files are still a good idea so that I know if I have covered the proper test scenarios or not.  When the prerelease work goes into INT, I plan to remove "-Repository Local" from all tests and run them as is, and my goal is for all of them to pass just like against Local.